Scripts/Python/islmRegisterNexusLink.py

```diff
@@ -108,6 +108,20 @@ def OnNotify(self,state,id,events):
                     vault.addChronicleEntry("GotLinkToKveerPublic",1,"yes")
                     PtDebugPrint("islmRegisterNexusLink.OnNotify(): Chronicle entry 'GotLinkToKveerPublic' not present, adding entry and setting to 'yes'")
                     PtSendKIMessage(kKILocalChatStatusMsg,PtGetLocalizedString("KI.Messages.NexusLinkAdded"))
+            # Hacking away for GoMe Pub
+            if stationName.value == "GoMePubNew":
+                entryName = "GotLinkToGoMePublic"
+                entry = vault.findChronicleEntry(entryName)
+                if entry is not None:
+                    entryValue = entry.chronicleGetValue()
+                    if entryValue != "yes":
+                        entry.chronicleSetValue("yes")
+                        entry.save()
+                        PtDebugPrint("islmRegisterNexusLink.OnNotify(): Chronicle entry 'GotLinkToGoMePublic' already added, setting to 'yes'")
+                else:
+                    vault.addChronicleEntry("GotLinkToGoMePublic",1,"yes")
+                    PtDebugPrint("islmRegisterNexusLink.OnNotify(): Chronicle entry 'GotLinkToGoMePublic' not present, adding entry and setting to 'yes'")
+                    PtSendKIMessage(kKILocalChatStatusMsg,PtGetLocalizedString("KI.Messages.NexusLinkAdded"))
```

Comment:
In a future update, I would like to change this script to make this and the K'veer case more generic. It will require simultaneous updates to K'veer and GoMePubNew, however. For now, we probably just need to go with this since it got released to MOULa without any proper code review.